dev/test_phase_wrap.py

```diff
@@ -0,0 +1,131 @@
+import os
```

Comment:
Okay sounds good.

Yeah I thought calling boxy.py would be the better way to implement these tests, but since we want to test each of the individual phase pre-processing steps, I wasn't sure how to best go about that.

I suppose it would be better to have the same code in matlab and python, we could then just compare the output directly rather than comparing each individual step.